client/api/contentElement.js

```diff
@@ -0,0 +1,13 @@
+import request from './request';
+
+const urls = {
+  resource: (repoId, id) => `/repositories/${repoId}/content-elements/${id}`
+};
+
+function patch(repoId, id, data) {
```

Comment:
Patch first argument should be entity (Rather than having repoId and id as separate args make first one be element and destructure id / repositoryId